v0/8-13-0_Deleted_profiles.markdown

```diff
@@ -2,9 +2,23 @@
 
 > Deleted profiles are archived, and can be restored if needed. 
 
-Once you have deleted a profile, they are archived in the system. This means that they and the {{work}} you have done with them will be included in {{report}}s. These profiles can be re-instated from the archive. To do this you will need to add them as a new user, (see section [8.4.0  Adding a profile](/help/index/v/{{version}}/p/8.4.0)) and click on their name as it appears in the duplicate box. When their profile appears, click on the undelete button on their profile 
+Once you have deleted a profile, they are archived in the system. This means that the work you have done with them will be included in reports. These profiles can be re-instated from the archive, my a user of Manager level and above.
```

Comment:
these profiles can be re-instated from the archive, my a user of Manager level and above.
these profiles can be re-instated from the archive, by an operator of Manager access level and above.